### ISSUE TITLE
Splitting a wall, does a number on a door that's been flipped. 

### ISSUE THREAD

**Issue by theoryshaw** [IfcOpenShell/IfcOpenShell#3673]
Splitting a wall, does a number on a door that's been flipped. 

https://github.com/IfcOpenShell/IfcOpenShell/assets/507113/6c4d29ec-f30a-47bf-8e90-736bf7bb94c1

**Comment by Moult** [comment]
Yeah I'm aware of quite a few flipping / mirroring bugs. Thanks for helping keep track :)
